embulk-cli/src/main/java/org/embulk/cli/EmbulkMigrate.java

```diff
@@ -236,7 +238,8 @@ public void copy(String sourceResourcePath, String destinationFileName)
         {
             Path destinationPath = this.basePath.resolve(destinationFileName);
             Files.createDirectories(destinationPath.getParent());
-            Files.copy(EmbulkMigrate.class.getClassLoader().getResourceAsStream(sourceResourcePath), destinationPath);
+            Files.copy(EmbulkMigrate.class.getClassLoader().getResourceAsStream(sourceResourcePath), destinationPath,REPLACE_EXISTING);
```

Comment:
Add a space or a newline after the comma. (If we use `StandardCopyOption.REPLACE_EXISTING`, a newline could be better.)